common/api/core-common.api.md

```diff
@@ -6025,10 +6025,17 @@ export interface ModelLoadProps {
     id?: Id64String;
 }
 
+// @public
```

Comment:
Is `public` the appropriate release tag? Note that adding a new value to an `enum` is a breaking API change - do you expect to add new drape targets in the future (e.g., draping on iModel geometry)?

Reply:
Yes, we expect new drape targets; thanks for the observation. Do you recommend @beta or @alpha?

Reply:
beta. Note that properties+functions inherit the release tags of their containing class/interface/namespace so you will need to add some @beta to those too.

Reply:
upgraded to beta.